src/node_jsvmapi.cc

```diff
@@ -1289,7 +1290,7 @@ napi_status napi_throw(napi_env e, napi_value error) {
       v8impl::V8LocalValueFromJsValue(error));
   // any VM calls after this point and before returning
   // to the javascript invoker will fail
-  return napi_ok;
+  return GET_RETURN_STATUS();
```

Comment:
Should this return `napi_ok`. As it is now, it will return `napi_pending_exception`, which is also correct I guess. But I think I'd expect a function that throws an exception to return a success code when the exception is successfully thrown.

Same for the following two functions.